src/components/InputandFilterPanel/InputandFilterPanel.styles.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from "styled-components";
+import searchIcon from "../../images/search.svg";
+
+export const StyledInputandFilterSection = styled.section`
+	padding: 20px 40px;
+	width: 100%;
+	display: flex;
+	flex-direction: column;
+	align-items: start;
+	@media screen and (min-width: 550px) {
```

Comment:
breakpoint should be defined in some config object

Reply:
I have defined breakpoints in theme. Hope it is okay.